### PR TITLE
hitbtc: add fetchMarginMode

### PR DESCRIPTION
**Pull request by sc0Vu**
```BASH
$ p hitbtc fetchMarginMode --swap
Python v3.11.4
CCXT v4.1.45
hitbtc.fetchMarginMode()
{ symbol: 'TRX/USDT:USDT', marginMode: 'cross' }

$ p hitbtc fetchMarginMode BTC/USDT:USDT
Python v3.11.4
CCXT v4.1.45
hitbtc.fetchMarginMode(BTC/USDT:USDT)
{ symbol: 'BTC/USDT:USDT', marginMode: 'isolated' }

# margin
$ p hitbtc fetchMarginMode
Python v3.11.4
CCXT v4.1.45
hitbtc.fetchMarginMode()
{ symbol: 'ETH/BTC', marginMode: 'isolated' }

$ p hitbtc fetchMarginMode  BTC/USDT '{"type":"margin"}'
Python v3.11.4
CCXT v4.1.45
hitbtc.fetchMarginMode()
{ symbol: 'BTC/USDT', marginMode: 'isolated' }
```

### CODE REVIEW

File: ts/src/hitbtc.ts [hunk 1]
```diff
@@ -2369,6 +2369,82 @@ export default class hitbtc extends Exchange {
...
+    async fetchMarginMode (symbol: string = undefined, params = {}) {
+        /**
+         * @method
+         * @name hitbtc#fetchMarginMode
+         * @description fetches margin mode of the user
+         * @param {string} symbol unified symbol of the market the order was made in
+         * @param {object} [params] extra parameters specific to the hitbtc api endpoint
+         * @returns {string} 'cross' or 'isolated'
+         */
+        await this.loadMarkets ();
```
Comment (carlosmiei) [inline ts/src/hitbtc.ts:2381]: @sc0Vu where's the test? 😅

File: ts/src/hitbtc.ts [hunk 2]
```diff
@@ -2369,6 +2369,82 @@ export default class hitbtc extends Exchange {
         }, market);
     }
 
+    async fetchMarginMode (symbol: string = undefined, params = {}) {
+        /**
+         * @method
+         * @name hitbtc#fetchMarginMode
+         * @description fetches margin mode of the user
+         * @param {string} symbol unified symbol of the market the order was made in
```
Comment (carlosmiei) [inline ts/src/hitbtc.ts:2377]: we're missing the @see

File: ts/src/hitbtc.ts [hunk 3]
```diff
@@ -2369,6 +2369,82 @@ export default class hitbtc extends Exchange {
...
+         * @param {object} [params] extra parameters specific to the hitbtc api endpoint
+         * @returns {string} 'cross' or 'isolated'
```
Comment (carlosmiei) [inline ts/src/hitbtc.ts:2379]: instead of returning a string, I think we could return the object with symbol and `marginMode`

Comment (sc0Vu) [reply ts/src/hitbtc.ts]: @carlosmiei I added MarginMode type in ts, are ts types also transpiled?

Comment (carlosmiei) [reply ts/src/hitbtc.ts]: @sc0Vu more or less, but I will check out

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**